test/testdata/dsl/t_struct/inexact.rb

```diff
@@ -0,0 +1,9 @@
+# typed: true
+
+class Inexact < T::InexactStruct
+  prop :foo, Integer
+  prop :bar, String
+end
+
+Inexact.new
+Inexact.new(foo: 3, bar: "hey") # error: Wrong number of arguments for constructor. Expected: `0`, got: `1`
```

Comment:
Why is this an error? I feel like this is a regression if this is an error.

Reply:
Are we missing something in our RBI in sorbet?

Reply:
Resolved offline: This has to do with the fact that we don't use sorbet-runtime in pay-server, so we never actually noticed this error was being emitted (which it shouldn't be.)